services/workflows-service/src/common/http-service/http-config.service.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpModule, HttpModuleOptions, HttpModuleOptionsFactory } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { AppLoggerService } from '../app-logger/app-logger.service';
+
+export const RETRY_DELAY_IN_MS = 1500; // extract to config, delay retry
+
+@Injectable()
+export class HttpConfigService implements HttpModuleOptionsFactory {
+  constructor(
+    private readonly configService: ConfigService,
+    private readonly logger: AppLoggerService,
+  ) {}
+
+  createHttpOptions(): HttpModuleOptions {
+    return {
+      timeout: this.configService.get('HTTP_TIMEOUT_IN_MS', 5000),
+      maxRedirects: this.configService.get('HTTP_MAX_REDIRECTS', 10),
```

Comment:
Does it not make sense to use `env` from @t3-env here?

Reply:
oh no this is the wrong way,
we should alway alway use ConfigService instead t3-env file...

As you can see here i already migrated the [env to our configService](https://github.com/ballerine-io/ballerine/blob/dev/services/workflows-service/src/app.module.ts#L64)
<img width="1112" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/584ddb40-6ebd-42f1-a438-6abc3a79b13a">